install/ui/test/README.md

```diff
@@ -1,18 +1,30 @@
 # WebUI Unit Tests
 
-How to run tests:
+## Prerequisites
 
-1. Run "autoreconf" command in the root of the repository
-2. Run "./configure" in the root of the repository
-3. Go to install/ui/src/libs/ directory
-4. Run "$ make" - this step generates loader.js which is necessary for load
+1. Run `autoreconf -i` command in the root of the repository
+2. Run `./configure` in the root of the repository
```

Comment:
Please recommend ``./autogen.sh`` instead of manually running autoreconf and configure. It's our recommended way to create Makefiles.